sky/serve/core.py

```diff
@@ -127,6 +159,8 @@ def up(
     controller_utils.maybe_translate_local_file_mounts_and_sync_up(task,
                                                                    path='serve')
 
+    tls_template_vars = _rewrite_tls_credential_paths(service_name, task)
```

Comment:
Should we rename the function? Seems not actually rewrite?
```suggestion
    tls_template_vars = _get_tls_template_vars(service_name, task)
```

Reply:
Actually, we are rewriting the `task.tls_credential` in the function. Renamed to `_rewrite_tls_credential_paths_and_get_tls_env_vars` now.